library/std/src/os/linux/net.rs

```diff
@@ -0,0 +1,70 @@
+//! Linux-specific extensions to primitives in the [`std::net`] module.
+//!
+//! [`std::net`]: crate::net
+
+use crate::io;
+use crate::net;
+use crate::sealed::Sealed;
+use crate::sys_common::AsInner;
+
+/// Os-specific extensions for [`TcpStream`]
+///
+/// [`TcpStream`]: net::TcpStream
+#[unstable(feature = "tcp_quickack", issue = "96256")]
+pub trait TcpStreamExt: Sealed {
+    /// Sets the value of the `TCP_QUICKACK` option on this socket.
+    ///
+    /// If set, acks are sent immediately, rather than delayed if needed in
+    /// accordance to normal TCP operation. This flag is not permanent, it
+    /// only enables a switch to or from quickack mode.  Subsequent operation
+    /// of the TCP protocol will once again enter/leave quickack mode
+    /// depending on internal protocol processing and other factors.
+    /// (since Linux 2.4.4)
```

Comment:
```suggestion
    /// Enable or disable `TCP_QUICKACK`.
    ///
    /// This flag causes Linux to eagerly send ACKs rather than delaying them. Linux may reset this flag after further operations on the socket.
    ///
    /// See [`man 7 tcp`](https://man7.org/linux/man-pages/man7/tcp.7.html) and [TCP delayed acknowledgement](https://en.wikipedia.org/wiki/TCP_delayed_acknowledgment) for more information.
```

(Modulo word-wrapping.)